protocol/rpcconsumer/rpcconsumer.go

```diff
@@ -327,7 +325,10 @@ func (rpcc *RPCConsumer) Start(ctx context.Context, options *rpcConsumerStartOpt
 
 	utils.LavaFormatDebug("Starting Policy Updaters for all chains")
 	for chainId := range chainMutexes {
-		policyUpdater, ok := policyUpdaters.Load(chainId)
+		policyUpdater, ok, err := policyUpdaters.Load(chainId)
+		if err != nil {
+			utils.LavaFormatFatal("could not load policy Updater for chain", err, utils.LogAttr("chain", chainId))
```

Comment:
one chain will crash all others, shouldnt be fatal

Reply:
In this PR I tried to maintain the flow as it was before as much as I could. This is one example - it was fatal in this case before my changes. What should it be instead?

Reply:
on the next line it errors out and continues, lets do that

Reply:
Fixed: 9c59e9ca81e33b17b0b310f4e25524d153c0483b